src/versionUtils.ts

```diff
@@ -96,11 +96,11 @@ export async function pingGithubUrl(urlStr: string): Promise<boolean> {
 }
 
 export function isOldTagNaming(tagName: string): boolean {
```

Comment:
What does this do if tagName is undefined coming in?

Reply:
I check it and it will not happen for now. All the calling functions have a check in place before calling this function OR hand over a fixed string value. This is of course just the current state which can change in future. If the tagName is undefined we will return false here which is not really nice but will fail in the calling method because we would search for a tag named 'v' :)